extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -333,7 +333,9 @@ export class AmpAnalytics extends AMP.BaseElement {
         `${TAG}: No friendly parent amp-ad element was found for ` +
         'amp-analytics tag with iframe transport.');
 
-    this.iframeTransport_ = new IframeTransport(this.getAmpDoc().win,
+    this.iframeTransport_ = new IframeTransport(
```

Comment:
@zhaoqin I'm wondering why this change was needed. in the case of inabox, `this.win` is always identical to `this.getAmpDoc().win`.